### DISCUSSION TITLE
The field `comment` in the JSON string is not defined in the `ApiTaskTemplateInfo` properties

### DISCUSSION THREAD

**Discussion by vadicamo** [dres-dev/Client-Examples#4] [Q&A]
Hi, 

I'm using the Java Client example (https://github.com/dres-dev/Client-Examples/blob/main/java/src/main/java/dev/dres/example/Client.java) 
but at line 64 (`currentRuns = runInfoApi.getApiV2ClientEvaluationList(sessionId);`) I  got the error  

` Error during request: 'The field `comment` in the JSON string is not defined in the `ApiTaskTemplateInfo` properties. JSON: {"templateId":"467b4c80-95ab-4c91-92e4-6d32847044f8","name":"lhe-kis-1","comment":"","taskGroup":"KIS","taskType":"Visual Known Item Search","duration":300}', exiting`

I'm using DRES 2.0.0-RC1 

Any suggestions?

Best regards,
Lucia

**Comment by lucaro** [comment]
Looks like a definition version conflict. There will be an RC2 soon, possibly already tomorrow, that will address this.
